src/Controller/ProxyController.php

```diff
@@ -244,7 +244,7 @@ public function distribution(string $package, string $version, string $ref, stri
      */
     public function downloads(Request $request): JsonResponse
     {
-        $contents = \json_decode($request->getContent(), true);
+        $contents = \json_decode($request->getContent(), true, 512, JSON_THROW_ON_ERROR);
```

Comment:
If it's rector, I think this rule should be skipped. If it's you, then you forgot to wrap it again after updating the PR. :stuck_out_tongue: